src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -40,10 +40,12 @@ object LocalBackend {
  * Handles both local Docker runs as well as local direct command line executions.
  */
 class LocalBackend extends Backend with LazyLogging {
+
   /**
    * Executes the specified command line, using the supplied lookup function for expression evaluation.
    * Returns a `Map[String, Try[WdlValue]]` of output names to values.
    */
+  // FIXME: During testing I found that "call: Call"'s Command is ignored. Is this correct?
```

Comment:
It was only an issue when I called executeCommand out of context of its usual operation. I had a mocked call object returning a different command to what I separately passed in as commandString